release-notes/opensearch-dashboards.release-notes-1.3.8.md

```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
```

Comment:
nit - there should only be one blank line between sections.

Reply:
Thanks for the comments! Address it as per suggestion.